x-pack/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -292,9 +301,8 @@ export class HeadlessChromiumDriver {
       }
 
       if (!allowed || !this.allowRequest(interceptedUrl)) {
-        logger.error(`Got disallowed URL "${interceptedUrl}", closing browser.`);
         this.page.browser().close();
-        throw new Error(`Received disallowed URL in response: ${interceptedUrl}`);
+        logger.error(getDisallowedOutgoingUrlError(interceptedUrl));
```

Comment:
Do you happen to know if this closes the browser and the underlying Rx stream? This was a security fix a few patches back, and if we happen to hit a disallowed request we need to terminate the session ASAP

Reply:
I do see the browser is still closing, just want to make sure that no URLs can be read once responded

Reply:
This is in the response handler callback, so this area is disconnected from the underlying Rx stream.

I think it should be ok to keep throwing the error here - I reverted this part of the PR https://github.com/elastic/kibana/pull/71989/commits/e33e4f98f6cc168a8a73ec97da989b0e492f69f2